radixdlt/src/main/java/com/radixdlt/counters/SystemCounters.java

```diff
@@ -70,6 +70,8 @@
 		NETWORKING_SENT_BYTES("networking.sent_bytes"),
 		NETWORKING_RECEIVED_BYTES("networking.received_bytes"),
 
+		NETWORKING_DROPPED_ERROR_RESPONSES("networking.dropped_error_responses"),
```

Comment:
Would be better as vertex_sync.dropped_error_responses, as networking.* is really the low-level stuff that talks IP.

Maybe @talekhinezh has some suggestions.

Reply:
Well, it was hard to choose since this counter is updated at the point where we converting low level transport messages into internal objects. Messages are dropped before they are converted, so selection of the proper group for counter even more complicated.

Reply:
Yes, I think this is fine for now and practically speaking does what we need to do. I think we leave design work around this area when we gather more information.